website/blog/2023-11-15-run-webassembly-wasm-workloads-windows-and-macos.md

```diff
@@ -0,0 +1,342 @@
+---
+title: Unlock WebAssembly on macOS & Windows
+description: Spinning a OCI container image containing a WebAssembly/WASM workload on macOS or Windows should be as simple as running any other OCI image.
+slug: wasm-workloads-on-macos-and-windows-with-podman
+authors: [benoitf]
+tags: [podman-desktop, wasm, wasi, WebAssembly]
+hide_table_of_contents: false
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Seamlessly run WebAssembly/wasm binaries on macOS and Windows
+
+You might heard excitement recently about WASM and WASI. Imagine a world where you can effortlessly run wasm binaries and distribute them using Open Container Initiative (OCI) container images – a singular image deployable across multiple architectures.
+
+Though the concept seemed straightforward, accomplishing this task proved to be quite challenging, particularly on macOS and Windows. The complexity comes from the additional virtual machine running Linux. This machine needs all dependencies and prerequisites being correctly setup.
```

Comment:
'all dependencies and prerequisites being correctly' -> 'all of the dependencies and prerequisites correctly'?